common/src/main/java/whocraft/tardis_refined/common/blockentity/device/ManipulatorCraftingRecipeItem.java

```diff
@@ -0,0 +1,23 @@
+package whocraft.tardis_refined.common.blockentity.device;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.level.block.Block;
+
+public class ManipulatorCraftingRecipeItem {
```

Comment:
This class could do with some documentation

Reply:
Documented.